src/CribblyBackend.DataAccess/Repositories/GameRepository.cs

```diff
@@ -10,6 +10,7 @@ namespace CribblyBackend.DataAccess.Repositories
     public interface IGameRepository
     {
         Task<Game> GetById(int Id);
+        Task<List<Game>> GetByTeamId(int id);
```

Comment:
I wonder if it would be best to define our abstract interfaces with abstract types? Specifically, here, maybe it would be beneficial to use `IEnumerable<Game>` instead of `List`, unless there's a specific reason we need `List`